public/v2/apps/mongodb-backup-s3.json

```diff
@@ -0,0 +1,134 @@
+{
+    "captainVersion": "2",
+    "documentation": "https://github.com/halvves/mongodb-backup-s3",
+    "displayName": "MongoDB Backup Amazon S3",
+    "description": "Runs mongodump to backup data using cronjob to an Amazon S3 bucket",
+    "dockerCompose": {
+        "services": {
+            "$$cap_appname": {
+                "environment": {
+                    "AWS_ACCESS_KEY_ID": "$$cap_aws_access_key_id",
+                    "AWS_SECRET_ACCESS_KEY": "$$cap_aws_secret_access_key",
+                    "BUCKET": "$$cap_bucket",
+                    "BACKUP_FOLDER": "$$cap_backup_folder",
+                    "MONGODB_HOST": "$$cap_mongodb_host",
+                    "MONGODB_PORT": "$$cap_mongodb_port",
+                    "MONGODB_USER": "$$cap_mongodb_user",
+                    "MONGODB_PASS": "$$cap_mongodb_pass",
+                    "MONGODB_DB": "$$cap_mongodb_db",
+                    "EXTRA_OPTS": "$$cap_extra_opts",
+                    "CRON_TIME": "$$cap_cron_time",
+                    "TZ": "$$cap_tz",
+                    "CRON_TZ": "$$cap_cron_tz",
+                    "INIT_BACKUP": "$$cap_init_backup",
+                    "INIT_RESTORE": "$$cap_init_restore",
+                    "DISABLE_CRON": "$$cap_disable_cron"
+                },
+                "image": "halvves/mongodb-backup-s3@sha256:e709db5e0fe5d44873ac5a385af214e951f2a0366a7c02b515ebf7c983658b0c",
+                "restart": "always"
+            }
+        },
+        "version": "2"
+    },
+    "instructions": {
+        "start": "Runs mongodump to backup data using cronjob to an s3 bucket",
+        "end": "MongoDB Backup S3 is ready"
+    },
+    "variables": [
+        {
+            "description": "Your aws access key id (for your s3 bucket)",
+            "id": "$$cap_aws_access_key_id",
+            "label": "AWS ACCESS KEY ID",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Your aws secret access key (for your s3 bucket)",
+            "id": "$$cap_aws_secret_access_key",
+            "label": "AWS SECRET ACCESS KEY",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Your s3 bucket.",
+            "id": "$$cap_bucket",
+            "label": "BUCKET",
+            "validRegex": "/^.+$/"
+        },
+        {
+            "description": "Name of folder or path to put backups (eg myapp/db_backups/)",
+            "id": "$$cap_backup_folder",
+            "defaultValue": "mongo_dumps/",
+            "label": "BACKUP_FOLDER"
+        },
+        {
+            "description": "The host/ip of your mongodb database",
+            "id": "$$cap_mongodb_host",
+            "label": "MONGODB HOST",
+            "validRegex": "/^.+$/",
+            "defaultValue": "localhost"
```

Comment:
This default value makes is easy for the user to just click on deploy and miss the fact that this is not right. Maybe remove the default value here, and just add some bit to the description. Like:
```
            "description": "The host/ip of your mongodb database. For CapRover hosted database, use srv-captain--your-mongodb-name",
```

Reply:
done